### PR TITLE
Update to support Laravel 5.3.*

### PR DESCRIPTION
**Pull request by Isfirs**
I added a version to support the newly released Laravel 5.3

Consider tagging for `1.1.0` too.

I didn't follow adding a new branch since this is only a patch for supported laravel versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ Create a webdav filesystem disk:
 	...
 	'webdav' => [
 	    'driver'   => 'webdav',
-	    'baseUri'  => 'https://mywebdavstorage.com',
-	    'userName' => 'pascalbaljetmedia',
-	    'password' => 'supersecretpassword,
+	    'baseUri'  => env('WEBDAV_HOST', 'https://mywebdavstorage.com',
+	    'userName' => env('WEBDAV_USERNAME', 'jdoe'),
+	    'password' => env('WEBDAV_PASSWORD', ''),
```
Comment (Isfirs) [inline README.md:40]: The `.env` file is the prefered store for secret data. This example reflects that.

File: composer.json [hunk 2]
```diff
@@ -14,11 +14,16 @@
             "email": "info@pascalbaljetmedia.com",
             "homepage": "https://www.pascalbaljetmedia.com",
             "role": "Developer"
+        },
+        {
+            "name": "Isfirs",
+            "email": "isfirs.pb@gmail.com",
+            "role": "Contributor"
```
Comment (Isfirs) [inline composer.json:21]: I added a patch. Just some notes for that.

File: composer.json [hunk 3]
```diff
@@ -14,11 +14,16 @@
...
         }
     ],
     "require": {
         "php" : "~5.6|~7.0",
-        "illuminate/filesystem": "5.0.*|5.1.*|5.2.*",
+        "illuminate/filesystem": "5.0.*|5.1.*|5.2.*|5.3.*",
```
Comment (Isfirs) [inline composer.json:26]: The new `Laravel` tag.

**Comment by pascalbaljet** [comment]
Fixed with release 1.0.2: https://github.com/pascalbaljetmedia/laravel-webdav/releases/tag/1.0.2

**Pull request closed without merging**